katas/helpers/alphabetObject.js

```diff
@@ -0,0 +1,38 @@
+const alphabet = {
+    a: 1,
+    b: 2,
+    c: 3,
+    d: 4,
+    e: 5,
+    f: 6,
+    g: 7,
+    h: 8,
+    i: 9,
+    j: 10,
+    k: 11,
+    l: 12,
+    m: 13,
+    n: 14,
+    o: 15,
+    p: 16,
+    q: 17,
+    r: 18,
+    s: 19,
+    t: 20,
+    u: 21,
+    v: 22,
+    w: 23,
+    x: 24,
+    y: 25,
+    z: 26,
+};
+
+const reversedAlphabet = (alphabetObj) => {
+
+    const reversedKeys = Object.keys(alphabetObj).reverse();
```

Comment:
ten reversedAlphabet nie musi być funkcją może być raz policzony, żeb za każdym razem nie przeliczać. I nazwa mówi że odwraca alfabet a jeszcze dodaje jakie znaki specjalne? Nie fajnie

Reply:
@maciejdudzinski hmm no tego konkretnie wymagała ta kata -> żeby te 3 znaki wziąć jeszcze pod uwagę.

Lepiej byłoby to rozbić na np. reversedAlphabet który zawiera w sobie tylko liter, i np. reversedAlphabetWithOtherChars który ma też te inne?

Reply:
@maciejdudzinski docommitowałem fixy